src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -210,13 +208,28 @@ public static void fuzzerInitialize(String[] args) {
               throw new Error("Not reached");
             })
             .collect(Collectors.toList());
+
     throwsDeclarations =
         Arrays.stream(targetExecutables)
             .collect(Collectors.toMap(method
                 -> method,
                 method
                 -> Stream.concat(Arrays.stream(method.getExceptionTypes()), alwaysIgnore.stream())
                        .toArray(Class[] ::new)));
+
+    fuzzerInitialize(methodReference, new Meta(targetClass), targetExecutables, null);
+  }
+
+  public static void fuzzerInitialize(
+      String methodReference, Meta meta, Executable[] targetExecutables, Object targetInstance) {
```

Comment:
We could check that all `targetExecutables` are defined in the same class and then use that class to construct the `Meta` object. This would reduce the parameter count and potentially catch implementation bugs.